lib/tasks/verify-javascript.js

```diff
@@ -18,22 +18,19 @@ async function lint (config) {
 		const hasGitignore = await fs.exists(gitignorePath);
 		const ignorePatterns = hasGitignore ? gitignore(await fs.readFile(gitignorePath)) : ['bower_components/', 'node_modules/'];
 
-		// Get eslint config from the component directory,
-		// or use the default if it does not exist.
-		const eslintPath = path.join(config.cwd, '/.eslintrc.js');
+		// Get eslint config from the component directory.
+		const eslintFile = '.eslintrc.js';
+		const eslintPath = path.join(config.cwd, eslintFile);
 		const hasEslint = await fs.exists(eslintPath);
-		const overrideConfigFile = hasEslint ? eslintPath : null;
+		if (!hasEslint) {
+			throw new Error(`Could not lint JavaScript. The component is missing lint configuration "${eslintFile}".`);
+		}
 
 		const eslint = new ESLint({
 			baseConfig: {
-				extends: ["origami-component"],
-				env: {
-					'browser': true,
-					'mocha': true
-				},
 				ignorePatterns,
```

Comment:
this no longer includes bower by default, I think I can make a PR to our eslint conf for this

Reply:
um no I'm not going there. We also want to ignore everything in .gitignore and I don't think that's possible through .eslintrc alone (there is a cli option tough)